### ISSUE TITLE
Make the help command a bit easier to consume

### ISSUE THREAD

**Issue by michael-robbins** [michael-robbins/emojirades-plusplus#67]
Currently it's a bit complicated, make it easier to consume, limit it's contents, or split it into 2 commands.

One being the complicated one, and the default the 'simple to understand' one.

**Comment by michael-robbins** [comment]
Even better once #68 is done, link to the online help!
